webpack.config.js

```diff
@@ -260,7 +292,7 @@ function getWebpackConfig() {
 					},
 				} ),
 		] ),
-		externals: [ 'electron' ],
+		externals: _.compact( [ externalizeWordPressPackages && wordpressExternals, 'electron' ] ),
```

Comment:
TIL [externals can be functions](https://webpack.js.org/configuration/externals/#function)